core/templates/dev/head/components/help_modal_directive.html

```diff
@@ -0,0 +1,60 @@
+<script type="text/ng-template" id="modals/userHelp">
+<div class="modal-body oppia-long-text protractor-test-welcome-modal">
+  <h1>Need Help with Something?</h1>
+  <p>
+    You can check out the resources in our help center or
+    take the tour of the exploration editor
+  </p>
+  <a href="https://oppia.github.io/#/" target="_blank" class="btn btn-success oppia-help-dismiss protractor-test-dismiss-welcome-modal" ng-click="goToHelpCenter()">Visit the Help Center</a>
+  <button class="btn btn-success oppia-welcome-start protractor-test-start-tutorial" ng-click="beginTutorial()">Take the Tour</button>
+</div>
+</script>
+<style>
+  .oppia-help-dismiss,
+  .oppia-help-modal .modal-body,
+  .oppia-help-modal .oppia-long-text p {
+    text-align: center;
+  }
+
+  .oppia-help-modal .modal-body {
+    height: 100%;
+    margin: 0 auto;
+    width: 90%;
+  }
+
+  .oppia-help-modal .modal-content {
+    border-radius: 0;
+    height: 100%;
+  }
+
+  .oppia-help-modal .modal-dialog {
+    height: 50%;
+    width: 40%;
+  }
+
+  .oppia-help-modal .in .modal-dialog {
```

Comment:
I tried combining them both ways but it looks like both are needed. The declaration with the .in class more accurately affects the modal placement within the window. When trying to just use .modal-dialog, it didn't work. When trying to use just .in, I couldn't affect the height of the modal.